rskj-core/src/test/java/co/rsk/core/bc/TransactionPoolImplTest.java

```diff
@@ -802,8 +825,8 @@ public void addTwoTransactionsOutOfOrderInNonceUsingAddTransaction() {
         Transaction tx1 = createSampleTransactionWithGasPrice(1, 0, 1000, 0, 1);
         Transaction tx2 = createSampleTransactionWithGasPrice(1, 0, 2000, 1, 2);
 
-        Assert.assertTrue(transactionPool.addTransaction(tx2).transactionWasAdded());
-        Assert.assertTrue(transactionPool.addTransaction(tx1).transactionWasAdded());
+        Assert.assertTrue(transactionPool.addTransaction(tx2).transactionsWereAdded());
+        Assert.assertTrue(transactionPool.addTransaction(tx1).transactionsWereAdded());
```

Comment:
I would make more assertions here about the return value of each addTransaction. Check that the first one returns ok queued with tx2,  the latter one should add ok pending with the list of the two tx in the right order (first tx1, then tx2).

Reply:
the test below (addTwoTransactionsOutOfOrderInNonceUsingAddTransactions), you didn't modify it on this PR, but should have assertions done to the return value as well

Reply:
also this `addTransaction` only retrieves an internal `TransactionPoolAddResult` instead of returning internal result+successors (successors are also in pending state now)

Reply:
I'll raise another issue to refactor those two methods into one

Reply:
#1286 https://github.com/rsksmart/rskj/issues/1286

Reply:
:ok_hand: :ok_hand: :ok_hand: